frontend/app/models/transportation-analysis.js

```diff
@@ -162,6 +162,13 @@ export default class TransportationAnalysisModel extends Model {
     return this.ratioFor('restaurantSqFt');
   }
 
+  // Fast food sq ft
+  @computed('project.commercialLandUse.[]')
+  get fastFoodSqFt() {
+    const fastFoodResturants = this.get('project.commercialLandUse').filter((r) => r.type === 'fast-food');
+    return fastFoodResturants.reduce((a, r) => a + r.grossSqFt, 0);
```

Comment:
Thanks! I decided to use more specific variable names for the accumulator and reducer.... but lmk if there's any objection and petitions for just `accumulator` and `currentElement`.
Will force push the new changes.

Reply:
I'm just saying you don't need to define another `fastFoodResturants` variable because this is all in one computed called `fastFoodSqFt`. 

If it feels like it's doing too much, you know the drill :)

Reply:
oh yeah, got that! Really good call.

I was just going on a tangent and pointing out that while I was refactoring I also renamed the `r` and `a, r` variables. Sorry that was probably confusing.